website/blog/2023-03-02-aluminium.md

```diff
@@ -0,0 +1,696 @@
+---
+author: Tomasz Godzik
+title: Metals v0.11.11 - Aluminium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.11.11, which brings in a couple
+of new features and improvements as well as stability fixes. Further releases
+and possibly 1.0.0 release will focus on providing a better stable development
+environment.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">141</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">108</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">9</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">TODO</td>
```

Comment:
- [ ] Just a reminder to add this in.

Reply:
Thanks! I totally forgot about that one.